### PR TITLE
feat(api): Implement OrganizationProcessingIssuesEndpoint (APP-966)

### PR DESCRIPTION
**Pull request by wedamija**
Implements OrganizationProcessingIssuesEndpoint, which is the org level version of
ProjectProcessingIssuesEndpoint. Refactored code to share the logic between the
two and improved tests.

### CODE REVIEW

File: src/sentry/api/endpoints/organization_processingissues.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        For each Project in an Organization, list its processing issues. Can
+        be passed `project` to filter down to specific projects.
+
+        :pparam string organization_slug: the slug of the organization.
+        :qparam array[string] project: An optional list of project ids to filter
+        to within the organization
+        :auth: required
+
+        """
+        projects = Project.objects.filter(id__in=self.get_project_ids(request, organization))
```
Comment (macqueen) [inline src/sentry/api/endpoints/organization_processingissues.py]: depending on which order these go in, one of us will need to update this to use https://github.com/getsentry/sentry/pull/11518

File: src/sentry/api/helpers/processing_issues.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+        last_seen = agg_results.get('last_seen')
+        data = {
+            'hasIssues': num_issues > 0,
+            'numIssues': num_issues,
+            'lastSeen': last_seen and serialize(last_seen) or None,
+            'resolveableIssues': project_resolveable.get(project.id, 0),
+            # XXX: Due to a bug in `find_resolved`, this was always returning
+            # False. It's unused in our frontend, so just defaulting to False
+            # so that we don't break any other consumers that expect this value.
```
Comment (macqueen) [inline src/sentry/api/helpers/processing_issues.py]: did you/should we file a jira ticket? it looks like the second value is used in a task so probably worth fixing even if it's not actually needed in the ui

**Review by macqueen — APPROVED** [review]

File: src/sentry/api/endpoints/organization_processingissues.py (hunk 1 above)
Comment (wedamija) [reply src/sentry/api/endpoints/organization_processingissues.py]: Just approved, you land first and I'll update.

File: src/sentry/api/helpers/processing_issues.py (hunk 2 above)
Comment (wedamija) [reply src/sentry/api/helpers/processing_issues.py]: Yeah, was planning on following up with a quick fix

**Review by markstory — COMMENTED** [review]
Looks good to me.

File: src/sentry/api/helpers/processing_issues.py (hunk 2 above)
Comment (wedamija) [reply src/sentry/api/helpers/processing_issues.py]: #11525

**Pull request merged**